tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -1095,22 +1095,141 @@ public function test_build_line_items( bool $itemized = false, array $expected_i
+		$product = null;
+		if ( ! empty( $cart_product_type ) ) {
+			$product = WC_Helper_Product::create_simple_product();
+			WC()->cart->add_to_cart( $product->get_id(), 1 );
+		}
+
+		$actual = WC_Stripe_Helper::is_adaptive_pricing_supported();
+
+		// Cleanup.
+		WC()->cart->empty_cart();
+		remove_filter( 'woocommerce_is_checkout', $is_checkout_filter );
+		WC_Stripe_Helper::update_main_stripe_settings( $stripe_settings );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WC_Stripe_Helper' in method 'test_is_adaptive_pricing_supported'. (undefined)

(StaticAccess)